src/App.tsx

```diff
@@ -9,6 +12,9 @@ const App = () => {
         <Route index element={<LandingPage />} />
         <Route element={<PageWithNavBar />}>
           <Route path="/placeholder" element={<h1>Placeholder</h1>} />
+          <Route path="/quiz" element={<QuizPage />} />
+          <Route path="/area51" element={<Area51Page />} />
+          <Route path="/*" element={<Error404Page />} />
```

Comment:
We discussed this on the call, but next time split this work out into three different PRs at least 😊